### DISCUSSION TITLE
Search by Asset ID?

### DISCUSSION THREAD

**Discussion by TheDjgamerProductions** [sysadminsmedia/homebox#374] [Q&A] [answered]
Hi There!

I am currently using the demo to see whether or not I should deploy this to my home lab.
I noticed that I am unable to search via Asset-ID, is this intentional or am I missing something?

Thanks!

**Comment by 101br03k** [comment, marked as answer]
Opening tips displays the wat to search by assed id: 
![image](https://github.com/user-attachments/assets/cadce305-7d9f-442a-992a-1aed5a52e8d0)


Or you can use a url that forwards you to the assed it, as is used by the qr code / label generator:  https://demo.homebox.software/a/000-001 wich can be found on https://demo.homebox.software/reports/label-generator
